operators/go/routeagent_controller.go

```diff
@@ -0,0 +1,165 @@
+	if err != nil {
+		if errors.IsNotFound(err) {
+			// Request object not found, could have been deleted after reconcile request.
+			// Owned objects are automatically garbage collected. For additional cleanup logic use finalizers.
+			// Return and don't requeue
+			return reconcile.Result{}, nil
+		}
+		// Error reading the object - requeue the request.
+		return reconcile.Result{}, err
+	}
+
+	// Define a new Pod object
+	pod := newPodForCR(instance)
+
+	// Set Routeagent instance as the owner and controller
+	if err := controllerutil.SetControllerReference(instance, pod, r.scheme); err != nil {
+		return reconcile.Result{}, err
+	}
+
+	// Check if this Pod already exists
+	found := &corev1.Pod{}
+	err = r.client.Get(context.TODO(), types.NamespacedName{Name: pod.Name, Namespace: pod.Namespace}, found)
+	if err != nil && errors.IsNotFound(err) {
+		reqLogger.Info("Creating a new Pod", "Pod.Namespace", pod.Namespace, "Pod.Name", pod.Name)
+		err = r.client.Create(context.TODO(), pod)
+		if err != nil {
+			return reconcile.Result{}, err
+		}
+
+		// Pod created successfully - don't requeue
+		return reconcile.Result{}, nil
+	} else if err != nil {
+		return reconcile.Result{}, err
+	}
+
+	// Pod already exists - don't requeue
+	reqLogger.Info("Skip reconcile: Pod already exists", "Pod.Namespace", found.Namespace, "Pod.Name", found.Name)
+	return reconcile.Result{}, nil
+}
+
+// newPodForCR returns a busybox pod with the same name/namespace as the cr
+func newPodForCR(cr *submarinerv1alpha1.Routeagent) *corev1.Pod {
+	labels := map[string]string{
+		"app": "submariner-routeagent",
+	}
+
+	// Create SecurityContext for Pod
+	// FIXME: Does this really need to be ALL, vs just NET_ADMIN? The current Helm-based deployment gives ALL.
+	//security_context_net_admin_cap := corev1.SecurityContext{Capabilities: &corev1.Capabilities{Add: []corev1.Capability{"NET_ADMIN"}}}
+	// FIXME: Seems like these have to be a var, so can pass pointer to bool var to SecurityContext. Cleaner option?
+	allow_privilege_escalation := true
+	privileged := true
+	// TODO: Verify all these permissions are needed
+	security_context_all_cap_allow_escal := corev1.SecurityContext{Capabilities: &corev1.Capabilities{Add: []corev1.Capability{"ALL"}}, AllowPrivilegeEscalation: &allow_privilege_escalation, Privileged: &privileged}
+
+	return &corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      cr.Name + "-pod",
+			Namespace: cr.Namespace,
+			Labels:    labels,
+		},
+		Spec: corev1.PodSpec{
+			Containers: []corev1.Container{
+				{
+					Name: "submariner-routeagent",
+					// TODO: Use var here
+					Image: "submariner-route-agent:local",
+					// FIXME: Should be entrypoint script, find/use correct file for routeagent
+					Command:         []string{"submariner-route-agent.sh"},
+					SecurityContext: &security_context_all_cap_allow_escal,
+					Env: []corev1.EnvVar{
+						{Name: "SUBMARINER_NAMESPACE", Value: cr.Spec.SubmarinerNamespace},
```

Comment:
You will also need 
SUBMARINER_CLUSTERCIDR,
SUBMARINER_SERVICECIDR

Reply:
Please take a look at the latest file for new additions - https://github.com/submariner-io/submariner-charts/blob/master/submariner/templates/route-agent-ds.yaml

Reply:
Okay, great, thanks for the tip. I raised this Jira to track: https://jira.coreos.com/browse/HYCLD-259

Reply:
I added these new RA CIDR vars to the Operator's deployment here: https://github.com/dfarrell07/submariner/pull/13

This allows PR/151 to be rebased on master and the Operator's Routeagent pod deployments to remain successful.

Thanks again! :pray: